### PR TITLE
Add exclude flag with glob

### PR DESCRIPTION
**Pull request by aviadhaham**

### CODE REVIEW

**Comment by aviadhaham** [comment]
I'm done
please check @orellazri

File: internal/app/app.go [hunk 1]
```diff
@@ -110,7 +116,7 @@ func (a *App) run(cCtx *cli.Context) error {
 		return fmt.Errorf("load datasources: %s", err)
 	}
 
-	inputFiles, err := a.compileGlob(cCtx.String("input"))
+	inputFiles, err := a.compileGlob(cCtx.String("input"), cCtx.String("exclude"))
```
Comment (orellazri) [inline internal/app/app.go:119]: don't pass it to the compileGlob function as it's not related.

call the compileGlob function again to get the exclude results and then calculate the diff here

**Review by orellazri — CHANGES_REQUESTED** [review]

File: internal/app/app.go (hunk 1 above)
Comment (aviadhaham) [reply internal/app/app.go:119]: :(

**Comment by aviadhaham** [comment]
fixed, please check again @orellazri

File: internal/app/parse.go [hunk 2]
```diff
@@ -112,3 +112,18 @@ func (a *App) compileGlob(pattern string) ([]string, error) {
 
 	return matches, nil
 }
+
+func (a *App) compileGlobExclusion(inputFiles []string, excludeFiles []string) []string {
```
Comment (orellazri) [inline internal/app/parse.go:116]: this function is not compiling globs, you should change its name to something like `excludeFilesFromInput` or `calculateInputExcludeDiff`, or `filterInputFilesWithExclusion`, etc.

File: internal/app/parse_test.go [hunk 3]
```diff
@@ -131,3 +131,36 @@ func TestCompileInputGlob(t *testing.T) {
 		filepath.Join(tmpSubdir, "input2.txt"),
 	}, files)
 }
+
+func TestCompileInputGlobWithExclusionGlob(t *testing.T) {
```
Comment (orellazri) [inline internal/app/parse_test.go:135]: change test name after changing function name

File: internal/app/parse_test.go [hunk 4]
```diff
@@ -131,3 +131,36 @@ func TestCompileInputGlob(t *testing.T) {
...
+	inputFiles, err := app.compileGlob(fmt.Sprintf("%s/%s", tmpDir, inputGlob))
+	require.NoError(t, err)
+	excludeFiles, err := app.compileGlob(fmt.Sprintf("%s/%s", tmpDir, excludeGlob))
+	require.NoError(t, err)
+	inputFiles = app.compileGlobExclusion(inputFiles, excludeFiles)
+
+	require.Equal(t, []string{filepath.Join(tmpDir, "3.txt")}, inputFiles)
+}
+
+func TestCompileGlobExclusion(t *testing.T) {
```
Comment (orellazri) [inline internal/app/parse_test.go:156]: change test name after changing function name

**Review by orellazri — CHANGES_REQUESTED** [review]

File: internal/app/parse.go (hunk 2 above)
Comment (aviadhaham) [reply internal/app/parse.go:116]: will do "filterExcludedFiles" @orellazri

**Comment by aviadhaham** [comment]
@orellazri renamed funcs, please check

File: README.md [hunk 5]
```diff
@@ -36,6 +36,7 @@ Run the `renderkit` command with the following arguments as either command-line
 | ---------------------- | ------------------------------------------------------------------------------ | ------ |
 | `config`               | Load configuration from YAML file                                              | string |
 | `input`                | The input glob to render                                                       | string |
+| `exclude`              | The glob pattern for files to skip during rendering                            | string |
```
Comment (orellazri) [inline README.md:39]: Use same description as usage from the CLI flag

**Review by orellazri — CHANGES_REQUESTED** [review]

**Pull request merged**